src/solvers/LevenbergMarquardt.jl

```diff
@@ -35,6 +35,10 @@ then the keyword `jacobian_tangent_basis` below is ignored
   a functor inheriting from [`StoppingCriterion`](@ref) indicating when to stop.
 * `expect_zero_residual` – (`false`) whether or not the algorithm might expect that the value of
   residual (objective) at mimimum is equal to 0.
+* `η` – parameter of the algorithm, the higher it is the more likely the algorithm will be
+  to reject new proposal points
```

Comment:
```suggestion
* `η` – Scaling factor for the sufficient cost decrease required to accept new proposal points. Allowed range: `0 < η < 1`.
```
How about this?

Reply:
That's what I find less vague :) thanks.

Reply:
Should we include the word `threshold` somehow.
The way I understand it is the threshold parameter for the subproblem to be judged as trustworthy.
from paper:
>Let η ∈ (0, 1) be a prefixed constant. If ρk ≥ η holds, we judge the subproblem is trustworthy

Reply:
The range helps a lot.

Reply:
A side question, if the range of η is (0,1) should ρk then also only be in that range?

Reply:
I'll leave the wording up to you, I can go change the docstring in `LevenbergMarquardtState` as well when everyone is happy.

Reply:
I think it's better to understand it in terms of cost decrease, i.e. instead of 
$\rho_k = \frac{f(x_{old}) - f(x_{new})}{denominator} \geq \eta$
we can have
$f(x_{old} )- f(x_{new}) \geq \eta * denominator$
which exposes the sufficient decrease nicely. I don't see any reason why ρk would also have to be in that range.

Reply:
I've added the word "threshold", it sounds like a good idea.

Reply:
I updated the docstring in LevenbergMarquardtState as well

Reply:
Great, thanks :+1: